std/json.d

```diff
@@ -230,6 +230,43 @@ struct JSONValue
         return store.floating;
     }
 
+    /***
+     * Value getter/setter for boolean stored in JSON.
+     * Throws: `JSONException` for read access if `this.type` is not
+     * `JSONType.true_` or `JSONType.false_`.
+     */
+    @property inout(bool) boolean() inout pure @safe
+    {
+        switch (this.type)
+        {
+            case JSONType.true_:
+                return true;
+            case JSONType.false_:
+                return false;
+            default:
+                throw new JSONException("JSONValue is not a boolean type");
+        }
```

Comment:
A three-branch switch seems a bit much to me.

```
if (type == JSONType.true_) return true;
if (type == JSONType.false_) return false;
throw ...
```

But... six vs. half a dozen.

Reply:
Done.